x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -112,3 +125,78 @@ include::{es-ref-dir}/setup/install/security-files-reference.asciidoc[leveloffse
 Additionally, when you use the enrollment token to connect {kib} to a secured {es} cluster, the HTTP layer CA certificate is retrieved from {es} and stored in the
 {kib} `/data` directory. This file establishes trust between {kib} and the {es}
 Certificate Authority (CA) for the HTTP layer.
+
+[discrete]
+[[stack-skip-auto-configuration]]
+=== Cases when security auto configuration is skipped
+When you start {es} for the first time, we try to automatically configure
+security for you. We run some checks to determine whether this is the first time
+that the node is starting. If any of those checks fail, there's a good
```

Comment:
```suggest
that the node is starting, if Security is not otherwise configured, and if the node configuration can be modified by the node startup process . If any of those checks fail
```
I'm not too happy with the above proposal, though. It's too verbose. I'll leave it up to you to tailor it.

Reply:
I'm going to turn this sentence into a bulleted list, such as:

> When you start {es} for the first time, the node startup process tries to automatically configure security for you. The process runs some checks to determine: 
> 
> * If this is the first time that the node is starting
> * Whether security is already configured
> * If the startup process can modify the node configuration